utils/message_html_to_component.jsx

```diff
@@ -70,11 +70,13 @@ export function messageHtmlToComponent(html, isRHS, options = {}) {
             shouldProcessNode: (node) => node.attribs && node.attribs[mentionAttrib],
             processNode: (node, children) => {
                 const mentionName = node.attribs[mentionAttrib];
+                const mentionHighlighted = node.parentNode.attribs.class && node.parentNode.attribs.class.includes('mention--highlight');
```

Comment:
This doesn't work unfortunately since the Markdown code doesn't properly highlight things ending with punctuation like `@joram.` on its own. Could we instead pass the `mentionHighlight` option into `messageHtmlToComponent`?

Reply:
I knew their would be an edge case that caused problems. Yeah that should be a good alternative.